web/app/js/components/TopRoutes.jsx

```diff
@@ -241,48 +267,53 @@ class TopRoutes extends React.Component {
     );
   }
 
-  renderResourceDropdown = () => {
+  renderResourceDropdown = (title, nameKey, typeKey, helperText) => {
     const { classes } = this.props;
     let { query, services, resourcesByNs } = this.state;
 
-    let key = "resource_name";
     let servicesWithPrefix = services
       .filter(s => s.namespace === query.namespace)
       .map(svc => `service/${svc.name}`);
     let otherResources = resourcesByNs[query.namespace] || [];
 
-    let dropdownOptions = servicesWithPrefix.concat(otherResources).sort();
-    let dropdownVal = _isEmpty(query.resource_name) || _isEmpty(query.resource_type) ? "" :
-      query.resource_type + "/" + query.resource_name;
+    let dropdownOptions = servicesWithPrefix
```

Comment:
i think the lines immediately above here need to reference `query.to_namespace` when creating the `To Resource` dropdown, otherwise the dropdown items are filtered by the wrong namespace:

![screen shot 2019-01-22 at 10 40 48 am](https://user-images.githubusercontent.com/236915/51557690-76181a80-1e32-11e9-8618-c7cd02927076.png)

Reply:
ahhh good catch!